### PR TITLE
hypershift: hypershift-private

### PR DESCRIPTION
**Pull request by LiangquanLi930**

### CODE REVIEW

File: ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml [hunk 1]
```diff
@@ -1,1217 +1,27 @@
 presubmits:
   openshift/release:
-  - agent: kubernetes
```
Comment (LiangquanLi930) [inline ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml:3]: @liangxia I found that the code below was deleted when I used the "make" command. Is this normal?

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-private-mgmt-p1-f7

File: ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml [hunk 2]
```diff
@@ -88,6 +88,11 @@ chain:
       oc wait --timeout=30m --for=condition=Available --namespace=clusters hostedcluster/${CLUSTER_NAME}
       echo "Cluster became available, creating kubeconfig"
       bin/hypershift create kubeconfig --namespace=clusters --name=${CLUSTER_NAME} >${SHARED_DIR}/nested_kubeconfig
+      
+      if [ "$(oc get hostedcluster/${CLUSTER_NAME -n clusters -o=jsonpath='{.spec.platform.aws.endpointAccess}')" == "Private" ]; then
```
Comment (liangxia) [inline ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml:92]: Unpaired '{' before `CLUSTER_NAME`

File: ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml (hunk 1 above)
Comment (liangxia) [reply ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml:3]: Sounds like a rebase issue

File: ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml (hunk 2 above)
Comment (LiangquanLi930) [reply ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml:92]: Thanks

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-private-mgmt-p1-f7

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-private-mgmt-p1-f7

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-private-disasterrecovery-p2-f14

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-private-mgmt-p1-f7

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-private-mgmt-p1-f7

**Pull request closed without merging**